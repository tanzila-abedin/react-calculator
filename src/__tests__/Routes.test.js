import renderer from 'react-test-renderer';
import Routes from '../Route';

test('Routes snapshot testing', () => {
  const tree = renderer.create(<Routes />).toJSON();
  expect(tree).toMatchSnapshot();
});
