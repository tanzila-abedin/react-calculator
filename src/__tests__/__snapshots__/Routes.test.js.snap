// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes snapshot testing 1`] = `
Array [
  <nav
    className="navbar"
  >
    <div
      className="links"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quotes
      </a>
    </div>
  </nav>,
  <div
    className="home container p-5"
  >
    <p
      className="text-center welcome-text"
      data-testid="welcomeText"
    >
      Welcome to Math Magician website
    </p>
  </div>,
]
`;
